### ISSUE TITLE
MetroRio (Rio de Janeiro - Brazil)

### ISSUE THREAD

**Issue by tchairolouco** [luu176/Metroflip#7]
Hey guys, here in Rio we use mifare classic 1k in our public/private metro system. I already have the keys cracked with my Flipper, is there interest in include this card in the app portifolio?

**Comment by luu176** [comment]
Yes, I'm interested in adding this card to Metroflip. First, is there an existing Flipper Zero NFC plugin to read the card's data? If not, feel free to contact me on Discord at: luu176. If a plugin is available, could you please provide the name of the public transport company?

**Comment by valdinojr** [comment]
Hi! The current MetroRio card can be read with the NFC Flipper Zero app,
and the keys from it I cracked with the provided MetroRio smartphone app
and some collecting nonces in the physical station. The company provides an
iOS / Android (most useful) app to check the card's info, recharge the
account and even use it to get the transportation itself.

This is the link from the Android

(
https://play.google.com/store/apps/details?id=com.thera.metrofacil&hl=pt_BR&pli=1
)

Right now I`m not with my Flipper Zero, but tomorrow I can send the .nfc
Flipper files that I have, with the keys and stuff. Thank you for your
attention!

Em ter., 10 de dez. de 2024 às 11:11, Luu ***@***.***>
escreveu:

**Comment by luu176** [comment]
There’s a bit of an issue—no one has fully figured out how these cards work yet since there’s no information available online. We’d need to reverse-engineer the card’s encoding, assuming there is any. I’d be happy to help you with this. What’s your preferred method of private communication? Do you use Discord or Signal? My Discord is luu176.

**Comment by valdinojr** [comment]
Yes, if you wish, contact me in Signal, @abacatina.30
